src/main_ghostty.zig

```diff
@@ -147,15 +151,9 @@ fn logFn(
         logger.log(std.heap.c_allocator, mac_level, format, args);
     }
 
-    switch (state.logging) {
-        .disabled => {},
-
-        .stderr => {
-            // Always try default to send to stderr
-            const stderr = std.io.getStdErr().writer();
-            nosuspend stderr.print(level_txt ++ prefix ++ format ++ "\n", args) catch return;
-        },
-    }
+    // Always try default to send to stderr
+    const stderr = std.io.getStdErr().writer();
+    nosuspend stderr.print(level_txt ++ prefix ++ format ++ "\n", args) catch return;
```

Comment:
Zig doesn't even have async anymore. The `nosuspend` keyword doesn't even do anything to my best knowledge.

Reply:
That was existing code.  I can remove it.

Reply:
The Zig 0.13.0 documentation still has nosuspend as does the example logFn in the std.log source so I'm going to leave this as is to be safe.